src/mame/mr/dribling.cpp

```diff
@@ -34,29 +32,17 @@
 
 ***************************************************************************/
 
-#include "emu.h"
 
+#include "emu.h"
+#include "machine/netlist.h"
+#include "netlist/devices/net_lib.h"
 #include "cpu/z80/z80.h"
 #include "machine/i8255.h"
-#include "machine/watchdog.h"
-
 #include "emupal.h"
+#include "speaker.h"
+#include "machine/watchdog.h"
 #include "screen.h"
-
-
-// configurable logging
-#define LOG_MISC     (1U << 1)
-#define LOG_SOUND    (1U << 2)
-#define LOG_PB       (1U << 3)
-
-//#define VERBOSE (LOG_GENERAL | LOG_MISC | LOG_SOUND | LOG_PB)
-
-#include "logmacro.h"
-
-#define LOGMISC(...)     LOGMASKED(LOG_MISC,     __VA_ARGS__)
-#define LOGSOUND(...)    LOGMASKED(LOG_SOUND,    __VA_ARGS__)
-#define LOGPB(...)       LOGMASKED(LOG_PB,       __VA_ARGS__)
```

Comment:
I removed the logging because the purpose was likely to log the events in order to implement the netlist. And I used that logging a lot to generate input files for ``nltool`` to run later. But I genuinely thought that they were put there only to help implement the netlist.

Reply:
The netlist isn’t fully implemented at this point, and configurable logging is harmless to leave in if it may help someone debugging in the future.